src/core/DialogTitle/index.tsx

```diff
@@ -22,8 +22,8 @@ const DialogTitle = forwardRef(function DialogTitle(
       {children || (
         <>
           {onClose && <CloseButton onClick={onClose} />}
-          <Title variant="h6">{title}</Title>
```

Comment:
because the v5 dialog title renders as an h2, nesting an h6 inside of it generates a11y errors. Removed h6 in favour of p tags